tests/phpunit/tests/theme/wpGetGlobalStylesForBlocks.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Tests wp_add_global_styles_for_blocks().
+ *
+ * @group themes
+ */
+class Tests_Theme_wpGetGlobalStylesForBlocks extends WP_UnitTestCase {
```

Comment:
I've bootstrapped this test from `Tests_Theme_wpGetGlobalStylesheet`. The important thing is `test_styles_for_blocks`. I haven't fully checked the theme setup, so there may be opportunities for simplification.

Reply:
```suggestion
class Tests_Theme_wpAddGlobalStylesForBlocks extends WP_UnitTestCase {
```